internal/client/client.go

```diff
@@ -65,17 +67,18 @@ func (c *Client) DoRequest(endpoint string, target interface{}) error {
 
 	http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: c.config.Bool("disable-ssl-verify")}
 	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
```

Comment:
It's pretty unlikely to actually come up, but there was technically a bug here. [If `NewRequest` throws an error, than `req` will be `nil`.](https://cs.opensource.google/go/go/+/refs/tags/go1.20.1:src/net/http/request.go;l=873), so the calls to `req.Header` which currently happen between `NewRequest` and it's error check will panic on a `nil` pointer.